dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCClient.scala

```diff
@@ -712,6 +785,36 @@ case class BinaryOutcomeDLCClient(
     spendingTxF
   }
 
+  def initiateMutualClose(
+      dlcSetup: SetupDLC,
+      sig: SchnorrDigitalSignature,
+      sendSigs: (SchnorrDigitalSignature, PartialSignature) => Future[Unit],
+      getMutualCloseTx: Future[Transaction]): Future[CooperativeDLCOutcome] = {
+    val fundingTx = dlcSetup.fundingTx
```

Comment:
log that we are beginning to attempt a mutual close of dlc, with the txids at minimum so we can follow this.